central_web_server/ui_files/species.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
This would be better as an object:
```
[
  {
    "name": "Common Crab",
    "image": "./img/common.png"
  },
  {
    etc...
  }
]
```
Then, to access the name, instead of doing `arr[index][0]` it would be `arr[index]["name"]`

Reply:
yeah fair enough